columnflow/columnar_util.py

```diff
@@ -993,8 +994,9 @@ def attach_behavior(
     By default, all subfields of *ak_array* are kept. For further control, *skip_fields* can contain
     names or name patterns of fields that are filtered.
     """
+
     if behavior is None:
-        behavior = getattr(ak_array, "behavior", None)
+        behavior = getattr(ak_array, "behavior") or coffea_nanoaod.behavior
```

Comment:
```suggestion
        behavior = getattr(ak_array, "behavior", None) or coffea.nanoevents.methods.nanoaod.behavior
```

Reply:
(just in case the behavior attribute is not even there instead of being None ... who knows)

Reply:
I actually tried this and it is indeed `None`, but I agree better safe than sorry :D